jormungandr-lib/src/interfaces/config/node.rs

```diff
@@ -0,0 +1,64 @@
+use crate::interfaces::{Log, Mempool};
+use serde::{Deserialize, Serialize};
+use std::net::SocketAddr;
+use std::path::PathBuf;
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct Rest {
+    pub listen: SocketAddr,
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct P2p {
```

Comment:
Should we also deduplicate the config structures in jormungandr?

Reply:
I think yes. There were some pr's after which test code had to be updated as well. I will try to do it in other pr